scripts/prepareForFirstWeekLaunch.ts

```diff
@@ -107,20 +187,16 @@ async function main() {
     0,
     MAX_UINT256,
   )
+
   // transfer lp tokens from deployer to signers
   await lpToken.transfer(signers[1].address, BIG_NUMBER_1E18)
   await lpToken.transfer(signers[2].address, BIG_NUMBER_1E18)
-  console.log(
-    "signer 1 bal: ",
-    (await lpToken.balanceOf(signers[1].address)).toString(),
-    "signer 2 bal: ",
-    (await lpToken.balanceOf(signers[2].address)).toString(),
-  )
   await lpToken.connect(signers[0]).approve(gauge.address, MAX_UINT256)
   await lpToken.connect(signers[1]).approve(gauge.address, MAX_UINT256)
   await lpToken.connect(signers[2]).approve(gauge.address, MAX_UINT256)
 
-  // Create max lock with 10M SDL for signer[0] to get boost
+  // Create max lock with 10M SDL for signer[1] to get boost
+  await sdl.approve(veSDL.address, MAX_UINT256)
```

Comment:
on line 208, could you lock less than the total amount of SDL contained in the wallet? That way we still have some SDL to lock manually and check balances etc